boards/arm/stm32_blackpill_v2/doc/index.rst

```diff
@@ -0,0 +1,163 @@
+| Interface  | Controller | Driver/Component                    |
++============+============+=====================================+
+| NVIC       | on-chip    | nested vector interrupt controller  |
++------------+------------+-------------------------------------+
+| SYSTICK    | on-chip    | system clock                        |
++------------+------------+-------------------------------------+
+| UART       | on-chip    | serial port                         |
++------------+------------+-------------------------------------+
+| GPIO       | on-chip    | gpio                                |
++------------+------------+-------------------------------------+
+| PINMUX     | on-chip    | pinmux                              |
++------------+------------+-------------------------------------+
+| FLASH      | on-chip    | flash                               |
++------------+------------+-------------------------------------+
+| SPI        | on-chip    | spi                                 |
++------------+------------+-------------------------------------+
+| I2C        | on-chip    | i2c                                 |
++------------+------------+-------------------------------------+
+| ADC        | on-chip    | ADC Controller                      |
++------------+------------+-------------------------------------+
+| USB OTG FS | on-chip    | USB device                          |
++------------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+        ``boards/arm/stm32_blackpill_v2/stm32_blackpill_v2_defconfig``
+
+Pin Mapping
+===========
+
+Available pins:
+---------------
+.. image:: img/Blackpill_Pinout.png
+      :align: center
+      :alt: Black Pill V2.0 Pinout
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_1 TX/RX : PA9/PA10
+- I2C1 SCL/SDA : PB8/PB9
+- SPI1 CS/SCK/MISO/MOSI : PA4/PA5/PB4/PA7 (Routed to footprint for external flash)
+- PWM_4_CH1 : PB6
+- PWM_4_CH2 : PB7
+- ADC_1 : PA1
+- USER_PB : PA0
+- USER_LED : PC13
+
+Clock Sources
+-------------
+
+The board has two external oscillators. The frequency of the slow clock (LSE) is
+32.768 kHz. The frequency of the main clock (HSE) is 25 MHz.
+
+The default configuration sources the system clock from the PLL, which is
+derived from HSE, and is set at 96MHz, which is the maximum possible frequency
+to achieve a stable USB clock (48MHz).
+
+Programming and Debugging
+*************************
+
+There are 2 main entry points for flashing STM32F4X SoCs, one using the ROM
+bootloader, and another by using the SWD debug port (which requires additional
+hardware). Flashing using the ROM bootloader requires a special activation
+pattern, which can be triggered by using the BOOT0 pin.
+
+Flashing
+========
+
+Installing dfu-util
+-------------------
+
+It is recommended to use at least v0.8 of `dfu-util`_. The package available in
```

Comment:
west provides a dfu_util runner, can you provide the runner configuration to be able to flash with west ?
(you can check 96b_carbon for valid instructions on that matter)

Reply:
I left another comment on this, but just following up here as well. Is adding

```
board_runner_args(dfu-util "--pid=0483:df11" "--alt=0" "--dfuse")

include(${ZEPHYR_BASE}/boards/common/dfu-util.board.cmake)
```
to board.cmake sufficient? Or is there another step. That much I did include, and west flash does work as expected.

Reply:
Ok, I oversaw the board.cmake file. This is fine then, but please update doc to reflect that west can be used, something like follows:
.. zephyr-app-commands::
:zephyr-app: samples/hello_world
:board: 96b_carbon
:goals: build flash

Reply:
Yep, I have already added pretty much exactly that! Thanks for the update, I will have an update pushed shortly.

Reply:
Fixed.